docs/_docs/includes.md

```diff
@@ -97,15 +103,18 @@ This include contains 5 parameters:
 You could also use `if` tags to safeguard scenarios where either the parameter is optional or where the author doesn't include the parameter:
 
 ```liquid
-{% raw %}<figure>{% if {{include.url}} %}<a href="{{include.url}}">
-{% endif %}<img src="{{include.file}}" {% if {{include.max-width}}
-%} style="max-width: {{include.max-width}}" {% endif %}
-alt="{{include.alt}}" />{% if {{include.url}} %}</a>{% endif %}
-{% if {{include.caption}} %}<figcaption>{{include.caption}}
-</figcaption>{% endif %}</figure>{% endraw %}
+{% raw %}<figure>
+   {% if {{ include.url }} %}<a href="{{ include.url }}">{% endif %}
+   <img src="{{ include.file }}" {% if {{ include.max-width }} %}
+   style="max-width: {{ include.max-width }}" {% endif %}
+   alt="{{ include.alt }}" />{% if {{ include.url }} %}</a>{% endif %}
+   {% if {{ include.caption }} %}
+   <figcaption>{{ include.caption }}</figcaption>
+   {% endif %}
+</figure>{% endraw %}
```

Comment:
I think it would be best if I just removed the section about conditionalizing the includes. I can take out this part: "You could also use `if` tags to safeguard scenarios where either the parameter is optional or where the author doesn't include the parameter:....."

I'm guessing the correct approach is to use the `default` filter, right?

If this really should be an error from Liquid, then yeah, the jekyll docs shouldn't promote bad code. 

I'll also omit the paragraph I added about spacing versus non-spacing and errors.